docs/ldml/tr35-info.md

```diff
@@ -1257,75 +1283,100 @@ The above information says that for default usage, in the US people use mile, fo
-* While in version 37 only 001 is used, in the future the data may contain others.
-* The fallback is: subdivision2 ⇒ subdivision1 ⇒ region/country ⇒ subcontinent ⇒ continent ⇒ world
-* Example:
-
-  | Region/subdivision | Code  |
-  | ------------------ | ----- |
-  | Blackpool          | gbbpl |
-  | England            | gbeng |
-  | United Kingdom     | GB    |
-  | Northern Europe    | 154   |
-  | Europe             | 150   |
-  | World              | 001   |
-
-* If there is an exact match for the region, but not for the usage,
-  * If the usage has multiple parts (eg land-agriculture-grain) drop the last part (eg land-agriculture)
-  * Repeat dropping the last part and trying the result (eg land)
-  * If you eliminate all of them, try usage="default".
-  * If there is no exact match for either one, try usage="default", region="001". That will always match.
-
-Once you have a list of `unitPreference` elements, find the applicable unitPreference. For a given category, usage, and set of regions (eg “US GB”), the units are ordered from largest to smallest.
-
+  
 * The geq item gives the value for the unit in the element value (or for the largest unit for mixed units). For example,
-  * `...geq="0.5">mile<...` means 0.9 kilometers
-  * `...geq="100.0">foot:inch<...` means 100 feet
+  * `...geq="0.5">mile<...` is ≥ 0.5 miles
+  * `...geq="100.0">foot-and-inch<...` is  ≥ 100 feet
 * If there is no `geq` attribute, then the implicit value is 1.0.
 * Implementations will probably convert the values into the base units, so that the comparison is fast. Thus the above would be converted internally to something like:
   * ≥ 804.672 meters ⇒ mile
-  * ≥ 30.48 meters ⇒ foot:inch
-* Search for the first matching unitPreference for the input measure. If there is no match (eg < 100 feet in the above example), take the last unitPreference. That is, the last unitPreference is effectively geq="0"
+  * ≥ 30.48 meters ⇒ foot-and-inch
+
+1. Search for the first matching unitPreference for the absolute value of the input measure. If there is no match (eg < 100 feet in the above example), take the last unitPreference. That is, the last unitPreference is effectively geq="0". In the above example, `<unitPreference regions="GB">yard</unitPreference>` is equivalent to `<unitPreference geq="0" regions="GB">yard</unitPreference>`
 
 For completeness, when comparing doubles to the geq values:
-* Negative numbers are treated as if they were positive.
-* _infinity_ is treated as being the largest possible value.
-* NaN is treated as the smallest possible value.
+* Negative numbers are treated as if they were positive, so in the above example -804.672 meters will format as "-0.5 mile".
+* _infinity_, NaN, and -_infinity_ match the largest possible value. Thus -∞ meters will format as "-∞ miles", not "-∞ yards".
```

Comment:
How are negative mixed units formatted? Probably similar to how we decided to format negative durations?

Reply:
Good point. I think the first unit should carry the negative sign.

-3 miles 150 feet
-3 hours 10 minutes
...

Reply:
Just added this, in the Mixed Units section.